internal/controllers/extension_controller.go

```diff
@@ -454,6 +452,23 @@ func (r *ExtensionReconciler) GenerateExpectedApp(o ocv1alpha1.Extension, bundle
 	return app
 }
 
+func (r *ExtensionReconciler) getInstalledVersion(ctx context.Context, extNN types.NamespacedName) (*bsemver.Version, error) {
```

Comment:
`extNN` is a little unclear to me (ah! NamespacedName). Would `extNamespacedName` be too verbose? Or maybe just `namespaceName`?